controllers/update_pods.go

```diff
@@ -154,6 +155,12 @@ func (updatePods) reconcile(ctx context.Context, r *FoundationDBClusterReconcile
 	return deletePodsForUpdates(ctx, r, cluster, adminClient, updates, logger)
 }
 
+func shouldRequeueDueToTerminatingPod(pod *corev1.Pod, cluster *fdbtypes.FoundationDBCluster, processGroupID string) bool {
+	return pod.DeletionTimestamp != nil &&
+		pod.DeletionTimestamp.Add(time.Duration(cluster.GetIgnoreTerminatingPodsSeconds())*time.Second).After(time.Now()) &&
```

Comment:
```suggestion
		pod.DeletionTimestamp.Add(cluster.GetIgnoreTerminatingPodsSeconds()*time.Second).After(time.Now()) &&
```

This will already return a `time.Duration`